src/flamenco/runtime/sysvar/fd_sysvar_cache.c

```diff
@@ -86,43 +86,52 @@ FD_SYSVAR_CACHE_ITER(X)
 
 /* Restore sysvars */
 
+# define X( type, name )                                                  \
```

Comment:
These macros can be kind of unclear to read through, but don't feel super strongly about these. Also some style nits here

Reply:
I just copied the existing style here in this file